test/graphviz_test.cpp

```diff
@@ -1,151 +1,192 @@
 // Copyright 2004-5 Trustees of Indiana University
+//
 
 // Use, modification and distribution is subject to the Boost Software
 // License, Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
 //
-// graphviz_test.cpp - Test cases for the Boost.Spirit implementation of a
-// Graphviz DOT Language reader.
+// graphviz_test.cpp - Test cases for the Graphviz DOT Language reader
 //
 
 // Author: Ronald Garcia
 
 #define BOOST_GRAPHVIZ_USE_ISTREAM
-#include <boost/regex.hpp>
 #include <boost/graph/graphviz.hpp>
 #include <boost/assign/std/map.hpp>
+#include <boost/assign.hpp>
 #include <boost/graph/adjacency_list.hpp>
 #include <boost/graph/compressed_sparse_row_graph.hpp>
-#include <boost/graph/graph_traits.hpp>
-#include <boost/tuple/tuple.hpp>
-#include <boost/property_map/dynamic_property_map.hpp>
 #include <boost/core/lightweight_test.hpp>
-#include <algorithm>
-#include <string>
+#include <boost/property_map/function_property_map.hpp>
 #include <iostream>
-#include <iterator>
-#include <map>
-#include <utility>
 #include <cmath>
 
-template<class T>
-class close_to {
-public:
-    explicit close_to(T f)
-        : f_(f) { }
+typedef std::string node_t;
+typedef std::pair< node_t, node_t > edge_t;
 
-    bool operator()(T l, T r) const {
-        return std::abs(l - r) <=
-            (std::max)(f_ * (std::max)(std::abs(l), std::abs(r)), T());
-    }
+typedef float Mass;
+typedef double Weight;
+typedef std::map< node_t, Mass > expected_masses_t;
+typedef std::map< edge_t, Weight > expected_weights_t;
```

Comment:
Personally, I think the type names should be more like `node_mass_map_t`, `edge_weight_map_t` and the variables should be named `expected_mass`, etc, but I'm not going to hold up the review over it.  :)